CHANGELOG.md

```diff
@@ -49,6 +49,9 @@ _None_
 - Throw syntax error on empty variable tags (`{{ }}`) instead `fatalError`.  
   [Ilya Puchka](https://github.com/ilyapuchka)
   [#263](https://github.com/stencilproject/Stencil/pull/263)
+- Fix for crashing range indexes when variable length is 1.  
```

Comment:
That's not the right place 😄 
Put it in the `Master` section under bug fixes.

Reply:
Fixed.